.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -1,6 +1,6 @@
 name: Bug report 🐛
 description: Create a bug report for MUI X.
-labels: ['status: waiting for maintainer']
+labels: ['status: waiting for maintainer', 'bug 🐛']
```

Comment:
To be mindful that not having the label was done purposefully. Not having the label removes all possible uncertainty: if the label is here, it's because a maintainer added it. I find it too often people open bugs while those are not bugs.

Reply:
I prefer to trust the author on this, since he was already given the option when entering the "New issue" page. Although I do agree that a lot of opened bug issues are asking for very specific implementation details, but I can still filter them out. 

From my experience so far more users opened under the correct category than vice versa

But yeah ... time will tell! :P